platforms/corda-ent/charts/notary-initial-registration/values.yaml

```diff
@@ -126,53 +168,18 @@ dataSourceProperties:
   # dbPort: 9101
   dbPort:
 
-notary:
-  # is the notary a validator Ex : validating: true
-  validating:
-
-# Name of the cenm services
-# Ex. idmanName: idman
-# Ex. networkmapName: networkmap
-cenmServices:  
-  idmanName: 
-  networkmapName:
-
-# Ex legalName: O=Notary,OU=Notary,L=London,C=GB
-legalName:
-# Ex emailAddress: dev@baf.com
-emailAddress:
-  
-rpcUsers:
-  # rpc user name Ex. notary
-  username:
-  # rpc user password Ex. notaryP
-  password: 
-
-# required parameter
-notaryPublicIP: 
-
-sleepTimeAfterError: 120
-sleepTime: 0
-
-# where is CENM service jar file stored
-jarPath: bin
-
-# where is CENM service configuration file stored
-configPath: etc
-
-healthCheckNodePort: 0
-
-jksSource: ""
-
+#############################################################
+#                       Settings                            #
+#############################################################
+# sleep time in seconds when error while registration
+# Ex. sleepTimeAfterError: 120
+sleepTimeAfterError:
+# custom sleep time in seconds
+sleepTime:
 healthcheck:

```

Comment:
Is healthCheck still needed? All occurrences should be able to be replaced by the vault values (`retries` & `retryInterval`)

Reply:
need for db health checks